lib/flutter_stories.dart

```diff
@@ -97,6 +100,12 @@ class Story extends StatefulWidget {
   ///
   final VoidCallback onFlashBack;
 
+  ///
+  /// Shows a close button at the right of screen below progress bar
+  /// and executes the callback when user taps on the close icon.
+  ///
+  final VoidCallback onClose;
```

Comment:
I'd strongly suggest you replace this callback with the builder of type `WidgetBuilder` and pass it as a child to `Positioned` down below. The good name would be e.g. `closeButtonBuilder`